<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManage</class>
 <widget class="QWidget" name="UserManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="searchLEd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>222</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>269</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>158</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>158</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>158</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
